projects/library/cohort-date-range/src/cohort-date-range.component.ts

```diff
@@ -213,6 +225,17 @@ export class TsCohortDateRangeComponent implements OnInit, OnDestroy {
   @Output()
   public readonly cohortDateRangeChanged = new EventEmitter<TsCohortDateRangeChanged>();
 
+  /**
+   * Define the max date for the starting date
+   */
+  @Input()
+  public startMaxDate: Date | undefined;
+
+  /**
+   * Define the min date for the starting date
+   */
+  @Input()
+  public startMinDate: Date | undefined;
```

Comment:
Might need to group all the `@Input()`... move these two above `@Output`.

Reply:
Fixed!